salesagility/SuiteCRM
# Fix 4921 alert nav

When using the search navigation bar in the top of the screen the text could become covered by the notification symbol.

## Description



The notification has been moved and altered to now adapt to the search bar and stay at a specific distance.

## How To Test This

1) Select Theme 7
2) Use the navigation bar at the top of the screen
3) The notification should not overlap now

## Types of changes


### Final checklist
